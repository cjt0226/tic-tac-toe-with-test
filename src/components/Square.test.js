import React from "react";
import { render, screen } from "@testing-library/react";
import Square from "./Square";
describe("Square",()=>{
    it("Should show the text",()=>{
        //Arrange
        render(<Square value="test_value"/>);
        //Act
        //Assert
        expect(screen.getByRole('button')).toMatchSnapshot();
        // screen.debug(screen.getByRole('button'));
    });
});