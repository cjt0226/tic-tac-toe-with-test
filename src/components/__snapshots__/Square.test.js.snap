// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square Should show the text 1`] = `
<button
  class="square"
>
  test_value
</button>
`;
